<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFrameBox</class>
 <widget class="QWidget" name="SendFrameBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="frameIdEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>frameIdLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="payloadEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="payloadEditLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>payloadEditLabel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dataFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>dataFrame</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="flexibleDataRateBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>flexibleDataRateBox</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="remoteFrame">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>remoteFrame</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="errorFrame">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>errorFrame</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>sendButton</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="extendedFormatBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>extendedFormatBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bitrateSwitchBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>bitrateSwitchBox</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
